packages/loader/container-loader/src/container.ts

```diff
@@ -880,20 +880,20 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
             // post summary here
             if (hasAttachmentBlobs) {
-                const appSummary: ISummaryTree = this.context.createSummary();
+                const appSummary: ISummaryTree = this.context.createSummary(new Map<string, string>());
```

Comment:
The argument is optional, I'd guess that means that there is no need to pass in empty map.
If we expect all callers to always pass it, should this arg be required?